config/webpacker.yml

```diff
@@ -48,6 +48,7 @@ default: &default
     - .gif
     - .jpeg
     - .jpg
+    - .mp3
```

Comment:
I guess it also needs to be removed here, no?

Reply:
@citizen428 Done